### PR TITLE
feat(osu!ppy): add buttons and multiLang

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

- This goes with https://github.com/PreMiD/Localization/pull/7285
- Add buttons with setting.
- Add multilang.
- Destructure variables.
## Acknowledgements
- [ ] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [ ] I linted the code by running `yarn format`
- [ ] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/O/osu ppy/presence.ts [hunk 1]
```diff
@@ -12,50 +61,65 @@ let beatmapTitle: string,
...
+	inputSelected: HTMLInputElement,
+	strings: Awaited<ReturnType<typeof getStrings>>,
+	oldLang: string = null;
 
 presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
-		largeImageKey:
-			"https://cdn.rcd.gg/PreMiD/websites/O/osu%20ppy/assets/logo.png",
-	};
+			largeImageKey: "logo",
```
Comment (theusaf) [inline websites/O/osu ppy/presence.ts:70]: Use an image URL over Discord asset name.

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/O/osu ppy/presence.ts (hunk 1 above)
Comment (DarkVillager) [reply websites/O/osu ppy/presence.ts:70]: > Use an image URL over Discord asset name.

I’ll resolve this tomorrow.

**Review by theusaf — CHANGES_REQUESTED** [review]
Resolve CI

**Review by theusaf — APPROVED** [review]

**Comment by theusaf** [comment]
Presence looks fine, awaiting localization merge.

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**